detox/test/e2e/25.statusbar.test.js

```diff
@@ -0,0 +1,170 @@
+
+function sleep(ms) {
+  return new Promise(resolve => setTimeout(resolve, ms));
+}
+
+describe('StatusBar Override', () => {
+  it('Time', async () => {
```

Comment:
Could you use [it.each](https://jestjs.io/docs/en/api#testeachtable-name-fn-timeout) instead of these numerous repetitions?

Reply:
@noomorph should I use one test for each so we can compare the screenshots?

Reply:
Fixed